requirements.txt

```diff
@@ -14,6 +14,6 @@ notebook
 opencv-python
 pandas
 pyarrow
-PyPDF2
+PyPDF2==2.5.0
 python-Levenshtein
```

Comment:
I'm not sure why I'm just noticing this, but in order to run any of the scripts locally, I needed to add the following two lines to the requirements.txt file: 
```
tabula-py
tabulate
```
There are two libraries used in the Durham data parsing, and the global `src/__init__.py` imports everything, so the whole module needs to be "Durham compatible."